tests/performance/density.go

```diff
@@ -100,6 +106,31 @@ var _ = SIGDescribe("Control Plane Performance Density Testing", func() {
 				waitRunningVMI(virtClient, vmCount+1, vmBatchStartupLimit)
 			})
 		})
+		**/
+
+		Context(fmt.Sprintf("[small] create a batch of %d running VMs", vmCount), func() {
+			It("should sucessfully create all VMS", func() {
+				By("Creating a batch of VMs")
+				createBatchRunningVMWithResourcesWithRateControl(virtClient, vmCount)
+
+				By("Waiting a batch of VMs")
+				waitRunningVMI(virtClient, vmCount+1, vmBatchStartupLimit)
```

Comment:
We create a "primer VMI" to initialize the metrics for the first test run.  `vmCount` should do.

Reply:
Ah of course, thanks!